### ISSUE TITLE
Footer Section

### ISSUE THREAD

**Issue by demetermarcell** [demetermarcell/bike-pals#4]
As a potential customer I want a section where I can get the contact details of the shop, so I can check their social media sites, call them or visit them in person.

#### The footer should have social media icons with accessibility aliases. These should bring the users to the social media sites of the shop in new tabs.
Social Media sites:

- Youtube
- Instagram
- Facebook
- TikTok

#### The footer should have an "Opening Hours" section with the following details:

- Mon-Sat:	10am-6pm
- Sunday: 	Closed

#### The footer should have an "Contact Us" section with the following details:

- Tel: +353-87-123-45-67
- Email:shop@bikepals.com
- Address: 12 Street, Dublin, D04AS12, Ireland

#### The footer should have an "Find Us" section with the following details:

- Google Maps iframe that shows the address of the shop.

## Mockups
![Image](https://github.com/user-attachments/assets/cde9a5ce-d679-43fa-8573-499d6f510538)

## Acceptance Criteria

- [x] The footer matches the mockups on all screen sizes
- [x] Google maps loads into the iframe showing the address of the shop.
- [x] Link tests are passed.

## Test Cases

1.
Given the user is on the footer section of the page.
When the user clicks on the Youtube logo.
Then a new tab opens with the Youtube channel of the shop.

2.
Given the user is on the footer section of the page.
When the user clicks on the Instagram logo.
Then a new tab opens with the Instagram channel of the shop.

3.
Given the user is on the footer section of the page.
When the user clicks on the Facebook logo.
Then a new tab opens with the Facebook page of the shop.

4.
Given the user is on the footer section of the page.
When the user clicks on the TikTok logo.
Then a new tab opens with the TikTok channel of the shop.
